src/Microsoft.TemplateSearch.TemplateDiscovery/Results/UnifiedPackCheckResultReportWriter.cs

```diff
@@ -92,7 +92,7 @@ private static TemplateDiscoveryMetadata CreateSearchMetadata(PackSourceCheckRes
                 additionalData[dataProducer.DataUniqueName] = dataProducer.Data;
             }
 
-            return new TemplateDiscoveryMetadata(TemplateInfo.CurrentVersion, templateCache, packToTemplateMap, additionalData);
+            return new TemplateDiscoveryMetadata("v1.0.0.4", templateCache, packToTemplateMap, additionalData);
```

Comment:
Is this line correct? I think you removed 1.0.0.4 in a different file, so not sure why we are using it.